packages/core/src/components/DateField/DateField.scss

```diff
@@ -46,6 +48,11 @@ Style guide: components.date-field
   width: 6ch; // provide additional spacing for browser UI
 }
 
+.ds-c-datefield__separator {
+  font-size: $h2-font-size;
+  line-height: $reset-line-height;
+  margin-bottom: 12px; // visually align with adjacent inputs
```

Comment:
I don't love a magic pixel number here, but the slash character in our font isn't flush with its line height. So with that, combined with the way the input/label groups are marked up, it seemed impractical to align this using a less manual method.

Reply:
Proposed solution to the magic: We can give it the same height and line-height and vertical margin as the text inputs, since we know this is always supposed to fit between the text inputs. Users of the design system can change the height and margins of the inputs, and that would throw off this magic number.

Reply:
@pwolfert -- something like the most recent change?